### PR TITLE
Geetest Captcha Support

### PR DESCRIPTION
**Pull request by IdekDude**

### CODE REVIEW

File: src/structs/2captcha.ts [hunk 1]
```diff
@@ -175,7 +175,7 @@ export class Solver {
             method: "userrecaptcha",
             ...this.defaultPayload
         }
-
+        console.log(utils.objectToURI(payload))
```
Comment (Furry) [inline src/structs/2captcha.ts:178]: Remove debugging statement

**Comment by Furry** [comment]
Hiya, I appreciate the PR! I'd be happy to merge it if you remove all the d.ts definition files, the compiled javascript files, the lock files, and anything else that wasn't directly related to your contribution please

**Comment by IdekDude** [comment]
Removed debugger statements and all files except for the 2captcha.ts

**Comment by IdekDude** [comment]
Anything that needs to be fixed in this PR, so it can get merged?

**Review by Furry — APPROVED** [review]

**Comment by Furry** [comment]
> Anything that needs to be fixed in this PR, so it can get merged?

Sorry for the wait, I'm not that experienced in Repository Management yet, and was unsure how to successfully merge this with the master branch. I just tried directly merging it, but it deleted the entire repository -- oops --, I reverted the PR and implemented your ``geetest`` solver code directly into the branch myself.

https://www.npmjs.com/package/2captcha version ``2.1.9`` is the newest release, containing the geetest implementation ^^

**Pull request merged**